cinema/urls.py

```diff
@@ -1,10 +1,42 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    movie_list,
+    movie_detail,
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet,
+)
+
+cinema_hall_list = CinemaHallViewSet.as_view({"get": "list", "post": "create"})
+cinema_hall_detail = CinemaHallViewSet.as_view(
+    {
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy",
+    }
+)
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
 
 urlpatterns = [
+    path("", include(router.urls)),
     path("movies/", movie_list, name="movie-list"),
     path("movies/<int:pk>/", movie_detail, name="movie-detail"),
```

Comment:
hmmm... 🤔 
are you sure?

Reply:
> hmmm... 🤔 are you sure?

seems like i've forgotten to delete those 2 lines